Sources/SQL/Core/main.swift

```diff
@@ -0,0 +1,141 @@
+extension Table {
+    public static func f(field: Field) -> DeclaredField {
```

Comment:
Yeah, I already commented about this at slack. It should be `field` if people want it shorter they can create an alias function in an extension.